extensions/amp-image-slider/0.1/amp-image-slider.js

```diff
@@ -137,7 +141,9 @@ export class AmpImageSlider extends AMP.BaseElement {
         '2 <amp-img>s must be provided for comparison');
 
     // TODO(kqian): remove this after layer launch
-    if (!isExperimentOn(this.win, 'layers')) {
+    if (!isExperimentOn(this.win, 'layers') ||
+      this.element.getOwner() !== null) {
```

Comment:
When have you observed that `this.element.getOwner()` is not null? Is this for `<amp-carousel>` only or are there other use cases for this too?